cypress/integration/visitorIsAbletoSeeTopTenMovies.feature.js

```diff
@@ -0,0 +1,18 @@
+describe('Visitor can see top 10 movies', () => {
+	before(() => {
+		cy.intercept('GET', 'https://worldwidenetflix.herokuapp.com/api/movies', {fixture: 'top10movies.json'} )
+	})
+	it('is expected to opend the main page', () => {
+		cy.visit('/')
+	})
+
+	it('is expected to show a list of top 10 global movies', () => {
+		cy.get('[data-cy=movie-container]').within(() => {
+			cy.get('[data-cy=movie-0]').within(() => {
+				cy.get('[data-cy=title-header]')
+			.should('contain', 'The Shawshank Redemption')
+			})
+			
+		})
+	})
+})
```

Comment:
Format this file, mae sure to use 2 spaces for indentation. 
And also, communicate with your backend team what the API will respond with if something goes wrong, sad path 👍🏻